src/Traversal/Sdk/Traversal.targets

```diff
@@ -26,9 +26,12 @@
     <EnableDefaultItems>false</EnableDefaultItems>
 
     <!--
-      TargetFramework is required for restore and used to default to .NET Framework v4.5.  However, Traversal projects don't specify a version so it needs to be defaulted here.
+      TargetFramework is required for restore. If the user specifies a version, use that. Otherwise, use the
+      NETCoreAppMaximumVersion so that the project acts as if it were part of the SDK. If for some reason that's also
+      not set, fallback to the SDK's old default of .NET Framework v4.5.
     -->
-    <TargetFramework Condition="'$(TargetFramework)' == ''">net45</TargetFramework>
+    <TargetFramework Condition=" '$(TargetFramework)' == '' And '$(NETCoreAppMaximumVersion)' != ''">net$(NETCoreAppMaximumVersion)</TargetFramework>
```

Comment:
What about something like `netstandard1.0` ?

Reply:
That would be an improvement over `net45`, but it doesn't fix the linked bug. 

Targeting old netstandard means we're still opt-ed out of modern defaults like defaulting to `Release` for `publish` and `pack`.

Reply:
Another alternative would be to find a solution for the project templates. Then we could default to the current SDK version in the template.

I also prefer that option a bit less since it means every Traversal would set the TFM which clutters things a bit, but it'd be less magical.